### ISSUE TITLE
What if swipe down to refresh would "Reload token data"?

### ISSUE THREAD

**Issue by colourfreak** [AlphaWallet/alpha-wallet-android#2020]
<img width="472" alt="Screenshot 2021-09-14 at 09 02 23" src="https://user-images.githubusercontent.com/51817359/133282716-91af24a7-1a3c-428a-b218-c4fd661a9c27.png">

This is already implemented on Android. You can go to Advanced settings and Reload Token Data. And I assume this is what users would expect to see, even though it takes ages sometimes to reload them all.

Thoughts?

**Comment by hboon** [comment]
Reload as in remove all the tokens and refetch/detect them? Doesn’t sound good then. Generally, pull down to refresh is just a refresh to hasten what would automatically be done regularly. Reload all, on the other hand is clearing the database and refetching from scratch — more of a mechanism to fix a bug, so it does belong to Settings (or a menu in Settings)

**Comment by JamesSmartCell** [comment]
> Reload as in remove all the tokens and refetch/detect them? Doesn’t sound good then. Generally, pull down to refresh is just a refresh to hasten what would automatically be done regularly. Reload all, on the other hand is clearing the database and refetching from scratch — more of a mechanism to fix a bug, so it does belong to Settings (or a menu in Settings)

Yes @hboon is correct - it's like a reset that support can advise if tokens aren't showing up usually because of a bug that got fixed. It's the 'dust off and nuke the entire site from orbit' option.

Swipe refresh currently initiates a reload from database and freshly starts the token update/discovery process as if you'd just re-started the app - other than tickers which still follow a 5 minute update cycle - mainly so we don't upset coingekko API too much.

Not sure I'd advise putting it onto the swipe refresh as it does trigger a set of API queries from Etherscan and Blockscout's various servers that can have a large footprint data footprint.
